.github/workflows/devbuild.yml

```diff
@@ -112,12 +112,25 @@ jobs:
         # https://doc-snapshots.qt.io/qt6-dev/linux.html
         #
         # In short, OpenGL libraries and headers are required. Without
-        # installing this package, cmake won't find out the correct lib path.
-        if [ "${{ matrix.os }}" == "ubuntu-22.04" ] ; then \
-          sudo apt-get -qy install \
-              qt6-3d-dev xvfb \
-              libgl1-mesa-dev
-        fi
+        # installing this package, cmake won't find the correct lib path.
+        # This has been replaced by the 'Install Qt' section below to manage qt6 versioning independently from the OS.
```

Comment:
You didn't pay attention to the line width (79-char).  The overly long lines make it harder to maintain.